sources/hubspot/settings.py

```diff
@@ -5,11 +5,11 @@
 STARTDATE = pendulum.datetime(year=2000, month=1, day=1)
 
 CRM_CONTACTS_ENDPOINT = (
-    "/crm/v3/objects/contacts?associations=deals,products,tickets,quotes&limit=100"
+    "/crm/v3/objects/contacts?properties={properties}associations=deals,products,tickets,quotes&limit=100"
 )
-CRM_COMPANIES_ENDPOINT = "/crm/v3/objects/companies?associations=contacts,deals,products,tickets,quotes&limit=100"
+CRM_COMPANIES_ENDPOINT = "/crm/v3/objects/companies?properties={properties}associations=contacts,deals,products,tickets,quotes&limit=100"
 CRM_DEALS_ENDPOINT = "/crm/v3/objects/deals?properties={properties}&limit=100"
 CRM_PRODUCTS_ENDPOINT = "/crm/v3/objects/products?limit=100"
-CRM_TICKETS_ENDPOINT = "/crm/v3/objects/tickets?limit=100"
+CRM_TICKETS_ENDPOINT = "/crm/v3/objects/tickets?properties={properties}limit=100"
```

Comment:
without ampersand in the url, you are producing the query (assoming props a,b) "properties=a,blimit=100" which will not be handled correctly